projects/addon-doc/utils/is-page-group.ts

```diff
@@ -1,7 +1,7 @@
-import type {TuiDocPage, TuiDocPageGroup} from '@taiga-ui/addon-doc/types';
+import type {TuiDocPageGroup, TuiDocRoutePage} from '@taiga-ui/addon-doc/types';
 
 export function tuiIsPageGroup(
-    page: TuiDocPage | TuiDocPageGroup,
+    page: TuiDocPageGroup | TuiDocRoutePage,
```

Comment:
Should we use `Route` suffix in all those types? In `TuiDocPages`, `TuiDocPageGroup`?